content/guides/integrations/sql/go_sql.md

```diff
@@ -0,0 +1,237 @@
+---
+title: "Go"
+date: 2018-07-16T14:42:23-07:00
+draft: false
+aliases: [/integrations/sql]
```

Comment:
nit: /integrations/sql -> /integrations/sql/go

Reply:
Oh, yes. Thanks, I'll update it.